app/models/dialog_field.rb

```diff
@@ -29,7 +29,7 @@ class DialogField < ApplicationRecord
                                   :message => "Field Name %{value} is reserved."}
 
   default_value_for :required, false
-  default_value_for(:visible, :allows_nil => false) { true }
```

Comment:
We need to `validates :visible, :presence => true` too

Reply:
oh right